### ISSUE TITLE
menclose tag results in odd speech for clearspeak

### ISSUE THREAD

**Issue by klobetime** [zorkow/speech-rule-engine#361]
The math `<math>
  <menclose notation="circle box">
    <mi> x </mi>
    <mo> + </mo>
    <mi> y </mi>
  </menclose>
</math>` results in "StartEnclose circle box x plus y EndEnclose" for mathspeak which makes sense, but for clearspeak we get "plus plus xy" which seems a bit off.  I can understand losing the circle box for clearspeak, but expected at least "x plus y" as a translation.

**Comment by zorkow** [comment]
Thanks for pointing this out. The mathspeak grammar is far more comprehensive than the clearspeak spec. I added rules for a number of missing cases (e.g., left indices, tensors), but clearly I missed a few. That should be an easy fix.

Up to 2.X SRE would take a default rule if it could find one, e.g., from the old ChromeVox rule set.
But since this led sometimes to odd interactions between rule sets I've removed that for 3.0.

So the result you see is SRE running out of rules to apply and simply returning the text content of the XML of its internal semantic tree.
